pages/TxKey.qml

```diff
@@ -133,12 +166,12 @@ Rectangle {
             MoneroComponents.Label {
                 id: soloTitleLabel2
                 fontSize: 24
-                text: qsTr("Check Transaction") + translationManager.emptyString
+                text: qsTr("Check Transaction / Reserve") + translationManager.emptyString
             }
 
             MoneroComponents.TextPlain {
                 text: qsTr("Verify that funds were paid to an address by supplying the transaction ID, the recipient address, the message used for signing and the signature.\n" +
-                           "For the case with Spend Proof, you don't need to specify the recipient address.") + translationManager.emptyString
+                           "For the case with Spend Proof, you don't need to specify the recipient address.\nTransaction is not needed for reserve proof.") + translationManager.emptyString
```

Comment:
Could you add this as a separate `qsTr()` so that we don't lose translations on the old string? Same with other strings, wherever possible.

Reply:
Thanks for reviewing @selsta! I have updated on commit 2bf0dd840f907b406271863540c8efe002b2cdc3 so that previous translations are not broken. Also removed the erroneous whitespace.